### DISCUSSION TITLE
【讨论】装饰器提案现在是什么状况? 未来估计会是什么结局?

### DISCUSSION THREAD

**Discussion by shawn-yee** [JSCIG/es-discuss#25] [General]
社区方案眼花缭乱,牵扯众多.
感觉会是成为无解之谜/有生之年系列.
看了几个node后端框架的装饰器写法的测试对比,会有20%左右性能下降.

**Comment by hax** [comment]
我没有参加最近的decorators会议， @legendecas 可能有所了解。

**Comment by hax** [comment]
装饰器提案已经进入 stage 3 一段时间，基本上就这样了。
